modules/agrvate/meta.yml

```diff
@@ -0,0 +1,40 @@
+name: agrvate
+description: write your description here
+keywords:
+  - sort
```

Comment:
```
keywords:
    - fasta
    - virulence
```

Reply:
@rpetit3 do you think that we should mention the `Staphylococcus aureus` as the keyword since this module is specifically focused on that species? 🤔

Reply:
That's a great question, I think yes because it will become helpful as more species-specific modules get added